docs/src/main/sphinx/installation/cli.rst

```diff
@@ -41,6 +41,27 @@ password value to avoid the prompt.
 Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
+External authentication - SSO
+-----------------------------
+
+The  ``--external-authentication`` option is used for browser-based SSO
+authentication, as detailed in :doc:`/security/oauth2`. With this configuration,
+the CLI displays a URL that you have to open in a web browser for authentication.
+
+The expected behaviour is as follows:
+
+ - Start the CLI with the ``--external-authentication`` option
+ - CLI starts and connects to Trino
+ - When first query is submitted, a message to open a browser with a specified URL is displayed by the CLI
```

Comment:
Awkward sentence. I suggest: "A message will appear in the CLI directing you to open a browser with a specified URL when the first query is submitted."